src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/model/entity/Visit.kt

```diff
@@ -81,18 +80,20 @@ data class Visit(
   @Temporal(TemporalType.TIMESTAMP)
   @Column
   var modifyTimestamp: LocalDateTime? = null,
+
+  @Column
+  var reference: String = "",
+
+  @Column
+  var active: Boolean
```

Comment:
Not sure if this adds anything @dhirajshettymoj , I did not have to use is any of the cases,... we could add it later when we need it?